mscore/inspector/inspectorAmbitus.cpp

```diff
@@ -144,18 +144,17 @@ void InspectorAmbitus::valueChanged(int idx)
             }
       }
 
-}
-
 //---------------------------------------------------------
-//   on updateRange clicked
+//   updateRange
 //---------------------------------------------------------
 
-void Ms::InspectorAmbitus::updateRange()
-{
+void InspectorAmbitus::updateRange()
+      {
       Ambitus* range = toAmbitus(inspector->element());
       range->updateRange();
-      range->layout();              // redo layout
-      setElement();                 // set Inspector values to range properties
-      valueChanged(AmbitusControl::TOPTPC);         // force score to notice new range properties
-}
+      range->layout();
```

Comment:
Nope, comments should stay OR maybe become actual. (depending on your reason for removing them)

This is especially true for the comment below. Because it's a SUPER STRANGE to see setters without arguments. I'm pretty sure that name of the function is wrong or some magic hidden here (or both). So a comment at least clarifies this a bit

Reply:
`setElement()` is widely used for inspectors, I don't think it's strange at all. It doesn't need any argument.

For tbe comments, I find them unnecessary for developers who're working on inspectors. `layout()` and `setElement()` are not hard to understand at all.

Reply:
> If you aren't familiar with it, please try to be ;-)

Bad argument :) It seems that you didn't catch my point.

Getters and setters are more than just a function. It's like a convention that provides property operation semantics. Like 'setTitle(..)' and 'getTitle() / title()'.

In this case, the name is confusing and you are forced to study the implementation. This, of course, has a bad effect on code readability, especially for new people.

Of course, now this is only an assumption, because now I have no way to get acquainted with the implementation, but i believe that the name of a function should be 'initElement() / init() / updateElement() / etc" and also i'm sure that it's not related to your PR, so... :)

Reply:
the `// redo layout` comment indeed seems prettyuseless (what else would a method called `layout() `do?), the other not so much, I buy @vpereverzev's argument here.

Reply:
Well, I agree the name `setElement()` should be changed too.